docs/guides/capacity-planning.md

```diff
@@ -44,7 +44,10 @@ Now, some rules of thumb:
  of series in ingesters is number of active series times the
  replication factor. This is with the default of 12-hour chunks - RAM
  required will reduce if you set `-ingester.max-chunk-age` lower
- (trading off more back-end database IO)
+ (trading off more back-end database IO).
+ There are some additional considerations for planning for ingester memory usage.
+    1. Memory increases during write ahead log (WAL) replay, [See Prometheus issue #6934](https://github.com/prometheus/prometheus/issues/6934#issuecomment-726039115). If you do not have enough memory for WAL replay, the ingester will not be able to restart successfully without intervention.
```

Comment:
**nit(optional):** it's a bit pedantic, however, write-ahead log should be hyphenated.

```suggestion
    1. Memory increases during write-ahead log (WAL) replay, [See Prometheus issue #6934](https://github.com/prometheus/prometheus/issues/6934#issuecomment-726039115). If you do not have enough memory for WAL replay, the ingester will not be able to restart successfully without intervention.
```